src/RazorConsole.Core/Components/Scrollable.razor

```diff
@@ -0,0 +1,76 @@
+﻿@namespace RazorConsole.Components
+
+@typeparam TItem
+
+@if (VisibleItems.Any())
+{
+    @ChildContent(new ScrollContext<TItem>(VisibleItems, HandleKeyDown, ScrollOffset, PagesCount))
```

Comment:
Can we also pass focus events to scroll context so the child control knows how to react to focus change. Otherwise user might not know if the scrollable get focused...